addons/website/static/src/js/widgets/fullscreen_indication.js

```diff
@@ -0,0 +1,50 @@
+odoo.define('website.widgets.FullscreenIndication', function (require) {
+'use strict';
+
+var Widget = require('web.Widget');
+
+var FullscreenIndication = Widget.extend({
+    xmlDependencies: ['/website/static/src/xml/website.xml'],
+    template: 'website.fullscreen_indication', 
+    init: function (parent) {
+        this.visible = false;
+        this.displayTime = 2000;
+        this._super(parent);
```

Comment:
If you're not using the arguments of a method you're overriding, don't declare them and simply call super with `...arguments`, this will allow the constructor you're overriding to change its arguments declaration without breaking your widget since they will just be passed along transparently